assets/src/web-stories-block/block/edit.js

```diff
@@ -70,23 +73,30 @@ function WebStoriesEdit({ attributes, setAttributes, className, isSelected }) {
     );
   }
 
-  return blockType === BLOCK_TYPE_LATEST_STORIES ? (
-    <LatestStoriesEdit attributes={attributes} setAttributes={setAttributes} />
-  ) : blockType === BLOCK_TYPE_SELECTED_STORIES ? (
-    <SelectedStoriesEdit
-      icon={icon}
-      attributes={attributes}
-      setAttributes={setAttributes}
-      isSelected={isSelected}
-    />
-  ) : (
-    <StoryEmbedEdit
-      icon={icon}
-      attributes={attributes}
-      setAttributes={setAttributes}
-      className={className}
-      isSelected={isSelected}
-    />
+  return (
+    <ConfigProvider config={config}>
+      {blockType === BLOCK_TYPE_LATEST_STORIES ? (
+        <LatestStoriesEdit
+          attributes={attributes}
+          setAttributes={setAttributes}
+        />
+      ) : blockType === BLOCK_TYPE_SELECTED_STORIES ? (
+        <SelectedStoriesEdit
+          icon={icon}
+          attributes={attributes}
+          setAttributes={setAttributes}
+          isSelected={isSelected}
+        />
+      ) : (
+        <StoryEmbedEdit
+          icon={icon}
+          attributes={attributes}
+          setAttributes={setAttributes}
+          className={className}
+          isSelected={isSelected}
+        />
+      )}
+    </ConfigProvider>
   );
```

Comment:
Might be a good opportunity to return early instead of using nested ternaries here, for readability.

```js
if (blockType === BLOCK_TYPE_LATEST_STORIES) {
	return ();
}

if ( ...) {
	return ();
}

return ();
```